src/components/editors/ExportEditor.js

```diff
@@ -41,6 +38,20 @@ const ExportEditor = () => {
 
   return (
     <Box p={2}>
+
+      <TextField
+        id="standard-basic"
+        label="Theme Name"
+        placeholder={themeName}
+        onChange={( { target: { value } } ) => {
+          // Empty string then set defualt (Overlay 2020-05-21 HH_MM_SS)
```

Comment:
```suggestion
          // Empty string then set default (Overlay 2020-05-21 HH_MM_SS)
```

Incorrect comment

Reply:
I should get a spell checker 😩

Reply:
I use Code Spell Checker `streetsidesoftware.code-spell-checker` 😄